pom.xml

```diff
@@ -122,7 +122,7 @@
     <nuxeo.features.version>9.3-SNAPSHOT</nuxeo.features.version>
     <nuxeo.addons.core.version>9.3-SNAPSHOT</nuxeo.addons.core.version>
 
-    <nuxeo.connect.client.version>1.6.1-SNAPSHOT</nuxeo.connect.client.version>
+    <nuxeo.connect.client.version>1.6.1</nuxeo.connect.client.version>
```

Comment:
As you're switching to a non-SNAPSHOT version you should also comment the `<exclude>org.nuxeo.connect:nuxeo-connect-client</exclude>` of the `maven-enforcer-plugin`.

Reply:
👍